tests/tools/onert_train/src/args.cc

```diff
@@ -230,9 +230,8 @@ void Args::Initialize(void)
         "1: CATEGORICAL_CROSSENTROPY\n")
     ("loss_reduction_type", po::value<int>()->default_value(0)->notifier([&] (const auto &v) { _loss_reduction_type = v; }),
         "Loss Reduction type\n"
-        "0: AUTO (default)\n"
-        "1: SUM_OVER_BATCH_SIZE\n"
-        "2: SUM\n")
+        "0: SUM_OVER_BATCH_SIZE(default)\n"
+        "1: SUM\n")
```

Comment:
(warning) 

Here, `onert_train` argument is modified. 
This could affect developer's local test-bed.